port_ocean/core/defaults/common.py

```diff
@@ -77,18 +78,28 @@ def deconstruct_blueprints_to_creation_steps(
     )
 
 
+def is_valid_dir(path: Path) -> bool:
+    return path.is_dir()
+
+
 def get_port_integration_defaults(
-    port_app_config_class: Type[PortAppConfig], base_path: Path = Path(".")
+    port_app_config_class: Type[PortAppConfig],
+    custom_defaults_dir: Optional[str] = None,
+    base_path: Path = Path("."),
 ) -> Defaults | None:
-    defaults_dir = base_path / ".port/resources"
-    if not defaults_dir.exists():
-        return None
-
-    if not defaults_dir.is_dir():
-        raise UnsupportedDefaultFileType(
-            f"Defaults directory is not a directory: {defaults_dir}"
+    fallback_dir = base_path / ".port/resources"
+
+    if custom_defaults_dir and is_valid_dir(base_path / custom_defaults_dir):
+        defaults_dir = base_path / custom_defaults_dir
+    elif is_valid_dir(fallback_dir):
+        defaults_dir = fallback_dir
```

Comment:
lets log that we are falling back, could be useful to track in NR whether integrations are tried to get initialized with missing defaults

Reply:
This is why I added the `logger.info(f"Loading defaults from {defaults_dir}")` log 😃

Reply:
that log doesn't actually state if we "fallback" to the default dir

Reply:
But we can see it from the `defaults_dir`, no?